### PR TITLE
Put teach.mozilla.org client_id in sawmill for Teach welcome emails

### PR DESCRIPTION
**Pull request by jbuck**

### CODE REVIEW

File: worker/send_new_user_email.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ module.exports = function(notifier_messager, mailroom) {
       return process.nextTick(cb);
     }
 
-    var mailTemplate = event.data.teach ? "user_created_teach" : "user_created";
+    var mailTemplate = event.data.client_id === teach_client_id ? "user_created_teach" : "user_created";
```
Comment (cadecairos) [inline worker/send_new_user_email.js]: what if both values are undefined :D

Comment (jbuck) [inline worker/send_new_user_email.js]: lol, teach emails for everyone

Comment (cadecairos) [inline worker/send_new_user_email.js]: should we worry about an edge case like that?

Comment (jbuck) [inline worker/send_new_user_email.js]: I just added a check above that sets it to false if it's a falsey value to handle this edge case

**Comment by cadecairos** [comment]
R+

**Pull request merged**